docs/integrations.rst

```diff
@@ -125,6 +125,13 @@ Consul
 .. automodule:: ddtrace.contrib.consul
 
 
+.. _datadog_lambda:
+
+Datadog Lambda
+^^^^^^^^^^^^^^
+.. automodule:: ddtrace.contrib.datadog_lambda
```

Comment:
The ddtrace datadog_lambda package was renamed. This will fix the failing build docs tests:
```suggestion
.. automodule:: ddtrace.contrib.aws_lambda
```